### PR TITLE
refactor(components): [select] use useNamespace

### PR DESCRIPTION
**Pull request by buqiyuan**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by sxzz — APPROVED** [review]

File: packages/components/select/src/select.vue [hunk 1]
```diff
@@ -88,8 +88,7 @@
               ref="input"
               v-model="query"
               type="text"
-              class="el-select__input"
-              :class="[selectSize ? `is-${selectSize}` : '']"
+              :class="[ns.e('input'), ns.is(selectSize, !!selectSize)]"
```
Comment (sxzz) [inline packages/components/select/src/select.vue:91]: **Suggestion:** we can change `is` function to
```ts
const is = (name: string, state = true) =>
    name && state ? `${statePrefix}${name}` : ''
```

and
```suggestion
              :class="[ns.e('input'), ns.is(selectSize)]"
```

**Review by sxzz — APPROVED** [review]

File: packages/components/select/src/select.vue (hunk 1 above)
Comment (sxzz) [reply packages/components/select/src/select.vue]: You need change `is` function as well.

Comment (buqiyuan) [reply packages/components/select/src/select.vue]: fixed！

File: packages/components/select/src/select.vue [hunk 2]
```diff
@@ -10,7 +10,8 @@
       v-model:visible="dropMenuVisible"
       placement="bottom-start"
       :teleported="compatTeleported"
-      :popper-class="`el-select__popper ${popperClass}`"
+      :append-to-body="popperAppendToBody"
+      :popper-class="`${ns.e('popper')} ${popperClass}`"
```
Comment (sxzz) [inline packages/components/select/src/select.vue:14]:
```suggestion
      :popper-class="[ns.e('popper'), popperClass]"
```

Comment (buqiyuan) [reply packages/components/select/src/select.vue]: OK

**Review by sxzz — APPROVED** [review]

**Review by sxzz — APPROVED** [review]

**Pull request merged**